docs/preact-testing-library/learn.mdx

```diff
@@ -15,10 +15,10 @@ If you're still hungry for more at this point than checkout:
 
 - The dom-testing-library:
   - [Introduction](intro.mdx)
-  - [Queries](dom-testing-library/api-queries.mdx)
+  - [Queries](queries/about.mdx)
   - [Firing Events](dom-testing-library/api-events.mdx)
   - [Async Utilities](dom-testing-library/api-async.mdx)
-  - [Helpers](dom-testing-library/api-helpers.mdx)
+  - [Helpers](dom-testing-library/api-custom-queries.mdx)
   - [FAQ](dom-testing-library/faq.mdx)
```

Comment:
We can probably remove this list entirely instead of trying to maintain the TOC in two places - just link to one page